### ISSUE TITLE
More comprehensive `git status` support

### ISSUE THREAD

**Issue by rouge8** [Canop/broot#686]
It would be great if broot's `git status` mode were a more complete replacement for `git status --short`. It'd probably require expanding the git column to two characters, but it would mean one fewer reason to leave broot 😉

**Comment by Canop** [comment]
Noted.
This isn't as trivial a change as it seems: broot currently only displays the present files (so a deleted file doesn't appear in git status).

**Comment by rouge8** [comment]
Yeah that makes sense, and I guess deleted seemed like the least important here. I’m more interested in the index vs working tree.
